pkg/config/config.go

```diff
@@ -64,6 +65,11 @@ const (
 	MonitorExecCgroupContainer = "container"
 )
 
+var (
+	kernelRROSupportOnce  sync.Once
+	kernelRROSupportError error
+)
+
```

Comment:
AFAICS this needs to be moved to config_linux.go, as it's only used from there.

Reply:
The idea was to share this amongst other implementations. However, I don't know if we will ever get FreeBSD one to work with. I will move it, indeed.

Reply:
Done.